src/components/core/pivotTable/AggregationsSubMenu.tsx

```diff
@@ -58,10 +58,12 @@ export default class AggregationsSubMenu extends React.Component<IAggregationsSu
         rowAttributeHeaders: Execution.IAttributeHeader[],
         afmAttributeHeaderIndex: number
     ): string {
-        return afmAttributeHeaderIndex === 0
-            ? this.props.intl.formatMessage({ id: 'visualizations.menu.aggregations.all-rows' })
-            // subtotals are presented differently to the user and in AFM (menu displays the previous attribute name)
-            : this.getPreviousAttributeName(rowAttributeHeaders, afmAttributeHeaderIndex);
+        if (afmAttributeHeaderIndex === 0) {
+            return this.props.intl.formatMessage({ id: 'visualizations.menu.aggregations.all-rows' });
+        }
+        const within = this.props.intl.formatMessage({ id: 'visualizations.menu.aggregations.within' });
+        const attributeName = this.getPreviousAttributeName(rowAttributeHeaders, afmAttributeHeaderIndex);
+        return `${within} ${attributeName}`;
```

Comment:
This is IMHO not a correct way how to do localization. `visualizations.menu.aggregations.within` string should be parametrized and `attributeName` value should be provided as parameter. This way the localization person can determine if the equivalent of "within" word should be at the start or end of the sentence.

`return intl.formatMessage({ id: 'visualizations.menu.aggregations.within' }, attributeName);`

(possibly the key of the message should be named differently than just `.within` after the change)

Reply:
fixed